benchmark_joins/src/join.rs

```diff
@@ -73,24 +76,36 @@ impl<'a> BlockNL<'a> {
     }
   }
 
+  fn get_effective_block_size(&self, num_records: usize, block_size: usize) -> usize {
```

Comment:
i think this should be effective_num_blocks still... block size is technically incorrect here since we are doing number of records/ number of records per block. let me know if you disagree

Reply:
Addressed in new commit